deployment/azure/arm/dittoInfrastructureDeployment.json

```diff
@@ -44,34 +92,59 @@
 
     },
     "variables": {
-        "clusterName": "[concat(parameters('uniqueSolutionPrefix'), 'dittoaks')]",
-        "accountName": "[concat(toLower(parameters('uniqueSolutionPrefix')), 'dittocosmos')]"
+        "dnsLabelPrefix": "[if(empty(parameters('dnsLabelPrefix')),concat(parameters('uniqueSolutionPrefix'),'ditto'),parameters('dnsLabelPrefix'))]",
+        "publicIPAddressName": "[if(empty(parameters('publicIPAddressName')),concat(parameters('uniqueSolutionPrefix'),'dittopip'),parameters('publicIPAddressName'))]",
+        "aksClusterName": "[if(empty(parameters('aksClusterName')),concat(parameters('uniqueSolutionPrefix'), 'dittoaks'),parameters('aksClusterName'))]",
+        "cosmosAccountName": "[if(empty(parameters('cosmosAccountName')),concat(parameters('uniqueSolutionPrefix'), 'dittocosmos'),parameters('cosmosAccountName'))]",
+        "vnetName": "[if(empty(parameters('vnetName')),concat(parameters('uniqueSolutionPrefix'), 'dittovnet'),parameters('vnetName'))]",
+        "aksSubnetName": "[if(empty(parameters('aksSubnetName')),concat(parameters('uniqueSolutionPrefix'), 'dittoakssubnet'),parameters('aksSubnetName'))]"
     },
     "resources": [{
             "condition": "[parameters('aks')]",
             "name": "dittoKubernetesDeployment",
             "type": "Microsoft.Resources/deployments",
             "apiVersion": "2018-05-01",
+            "dependsOn": [
+                "[concat('Microsoft.Resources/deployments/', 'dittoVnetDeployment')]"
+            ],
             "properties": {
                 "mode": "Incremental",
                 "templateLink": {
-                    "uri": "https://raw.githubusercontent.com/eclipse/ditto/master/deployment/azure/arm/templates/kubernetesDeploy.json"
+                    "uri": "https://raw.githubusercontent.com/kaizimmerm/ditto/aksvnet/deployment/azure/arm/templates/kubernetesDeploy.json"
```

Comment:
Is it intended to use your `kaizimmerm/ditto` fork here?

Reply:
Yes and no, usually @ghandim wants to test this but it won't work until the PR is merged with the eclipse repo URI. I will create a second commit later that changes this to the final URI.